pkg/sql/colexec/auto_incr.go

```diff
@@ -40,16 +41,20 @@ type AutoIncrParam struct {
 	rel     engine.Relation
 	ctx     context.Context
 	proc    *process.Process
+	dbName  string
+	tblName string
 	colDefs []*plan.ColDef
 }
 
-func UpdateInsertBatch(e engine.Engine, db engine.Database, ctx context.Context, proc *process.Process, ColDefs []*plan.ColDef, bat *batch.Batch, tableID string) error {
+func UpdateInsertBatch(e engine.Engine, db engine.Database, ctx context.Context, proc *process.Process, ColDefs []*plan.ColDef, bat *batch.Batch, tableID, dbName, tblName string) error {
 	incrParam := &AutoIncrParam{
 		eg:      e,
 		db:      db,
```

Comment:
If the db is a dead code, please remove it.

Reply:
sure